### PR TITLE
#182985522-SETUP SASS PROCESSING ON THE CLIENT

### PR DESCRIPTION
**Pull request by ChrisOnsando**
#### What does this Pull Request do?
- Install and setup sass processing on the client
#### Description of Task to be completed?
- Install and setup sass
- Create a sass folder structure
#### How should this be manually tested?
- N/A
#### Any background context you want to provide?
- Install and setup sass pre-processing on the application to enables the use of both .scss and .css files 
#### What are the relevant pivotal tracker story?
- (182985522)-https://www.pivotaltracker.com/story/show/182985522
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A

### CODE REVIEW

**Review by narravabrion — APPROVED** [review]

**Review by KaigaMunene — APPROVED** [review]

**Review by MwanikiN — APPROVED** [review]
Everything looks good.

**Comment by dalienst** [comment]
Reviewed

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "sass": "^1.54.4",
```
Comment (mainamuriuki274) [inline package.json]: should be a dev dependency

File: src/index.jsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
+import './index.scss';
```
Comment (mainamuriuki274) [inline src/index.jsx]: * rename src/sass put it in assets and then rename sass to styles
* remove this file src/index.scss
* only import the main style file from src/assets/styles here
* add the other folders and of styles for the 7 to 1 style

**Review by mainamuriuki274 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ChrisOnsando) [reply package.json]: Okay

File: src/index.jsx (hunk 2 above)
Comment (ChrisOnsando) [reply src/index.jsx]: Made changes

File: src/assets/styles/layout/_index.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+@import 'navbar'
```
Comment (mainamuriuki274) [inline src/assets/styles/layout/_index.scss]: add empty line after this

File: src/assets/styles/pages/_index.scss [hunk 4]
```diff
@@ -0,0 +1,2 @@
+@import 'home';
+@import 'login';
```
Comment (mainamuriuki274) [inline src/assets/styles/pages/_index.scss]: add empty line after this

File: src/assets/styles/themes/_index.scss [hunk 5]
```diff
@@ -0,0 +1 @@
+@import 'theme';
```
Comment (mainamuriuki274) [inline src/assets/styles/themes/_index.scss]: add empty line after this

File: src/assets/styles/vendors/_index.scss [hunk 6]
```diff
@@ -0,0 +1 @@
+@import 'bootstrap'
```
Comment (mainamuriuki274) [inline src/assets/styles/vendors/_index.scss]: add empty line after this

**Review by mainamuriuki274 — CHANGES_REQUESTED** [review]

**Review by mainamuriuki274 — APPROVED** [review]

**Pull request merged**